src/components/BottomBar/BottomBar.js

```diff
@@ -42,9 +53,62 @@ export const BottomBar = () => {
 
   const selectedNote = useSelector(state => state.notesMindMap.selectedNote);
 
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const open = Boolean(anchorEl);
+
+  const handleClick = event => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
```

Comment:
It's not obvious why ` setAnchorEl(null);` will close the menu.
I would prefer
```
const handleMenuClose = () => {
    setIsMenuOpen(false);
}
```

Reply:
Agree with you. But I achieved more transparency in another way